mne/viz/_brain/_brain.py

```diff
@@ -2559,7 +2560,8 @@ def add_foci(self, coords, coords_as_verts=False, map_surface=None,
 
     @verbose
     def add_sensors(self, info, trans, meg=None, eeg='original', fnirs=True,
-                    ecog=True, seeg=True, dbs=True, verbose=None):
+                    ecog=True, seeg=True, dbs=True, max_dist=0.004,
```

Comment:
Any thoughts on the necessity of the `'auto'` default as suggested in https://github.com/mne-tools/mne-python/pull/11437#issuecomment-1413949398 ?

Reply:
I don't think it's necessary, why would you plot the normal positions on an inflated brain? I don't see the use case

Reply:
So sEEG and ECoG do not need different `max_dist`?

Though thinking about it again I guess we can add that later when we add ECoG support so no need to change it here

Reply:
You can pass picks so you could use different distances